reactjs-films-homework/src/pages/MovieDetails/MovieDetails.tsx

```diff
@@ -0,0 +1,398 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
looks strange, we didn't expect disable lint, call me and let find solution together

Reply:
forgot to delete